.github/workflows/linting.yaml

```diff
@@ -13,7 +13,7 @@ jobs:
         uses: psf/black@stable
         with:
           src: "./apax"
-          version: "22.10.0"
+          version: "22.12.0"
```

Comment:
use pre-commit.ci now that the repo is public

Reply:
Will this be part of this PR or a separate one? I looked over it and the rest looks good for me.

Reply:
> Will this be part of this PR or a separate one? I looked over it and the rest looks good for me.

separate